docs/src/pages/IOSTextInputPage/IOSTextInputPage.js

```diff
@@ -79,15 +76,11 @@ const customSections = [
   },
 ];
 
-const IOSTextInputPage = ({ ...rest }) => (
-  <DocsPageBuilder
-    title="Text Input"
-    components={components}
-    readme={readme}
-    customSections={customSections}
-    iosDocId="BPKTextInput"
-    showMenu
-    {...rest}
+const IOSTextInputPage = () => (
+  <IOSComponentPage
+    screenshots={components}
+    readme={readme + textViewReadme}
```

Comment:
Interested to see how this runs locally

Reply:
I did the same thing on some of the Android pages in the previous PR. Looks fine!